src/components/settings.js

```diff
@@ -0,0 +1,148 @@
+
+import React,{ Component } from 'react';
+import {GridList, GridTile} from 'material-ui/GridList';
+import IconButton from 'material-ui/IconButton';
+
+import ActionDone from 'material-ui/svg-icons/action/done';
+import EditorModeEdit from 'material-ui/svg-icons/editor/mode-edit';
+
+import Avatar from 'material-ui/Avatar';
+import RaisedButton from 'material-ui/RaisedButton';
+
+import HeaderNav from './headerNav';
+
+import Paper from 'material-ui/Paper';
+
+import TextField from 'material-ui/TextField';
+import '../css/index.css'
+
+const styles = {
```

Comment:
move to css

Reply:
Can i please have permission to have that in the settings.js since it is a simple css and i wont be reusing it in any other component